modules/serverless-rn-1-24-0.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-24-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.24.0
+
+{ServerlessProductName} 1.24.0 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1.24.0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.3.
+* {ServerlessProductName} now uses Knative Eventing 1.3.
+* {ServerlessProductName} now uses Kourier 1.3.
+* {ServerlessProductName} now uses Knative `kn` CLI 1.3.
+* {ServerlessProductName} now uses Knative Kafka 1.3.
+* The `kn func` CLI plug-in now uses `func` 0.24.
+
+* Init containers support for Knative services is now generally available (GA).
+* FIPS is now disabled for the `KafkaBroker`, `KafkaSource` and `KafkaSink` components.
+
+ifdef::openshift-enterprise[]
+* You can now use the link:serverless/integrations/serverless-cost-management-integration.html#serverless-cost-management-integration_serverless-cost-management-integration[cost management service] with {ServerlessProductName}.
+endif::[]
+
+[id="fixed-issues-1.24.0_{context}"]
+== Fixed issues
+
+* The security issue, where a malicious actor could do a Denial-of-Service (DOS) attack on a server using the `InstrumentHandlerCounter` function in `prometheus/client_golang`, has been fixed. (CVE-2022-21698)
```

Comment:
There were a few more security issues fixed in this release. If we're mentioning one of them we should maybe list all? Look at [the list](https://issues.redhat.com/issues/?jql=project%20in%20(SRVCLI%2C%20SRVKE%2C%20SRVKS%2C%20SRVCOM%2C%20SRVOCF)%20AND%20issuetype%20%3D%20Bug%20AND%20fixVersion%20%3D%201.24.0)

Reply:
The set of documented release notes is the set of issues linked to the release notes Jira. Can you folks choose what else needs to be documented and link it to SRVCOM-1840 please?

The blank space at the top appears only in previews, so it's good. 👍

Reply:
As discussed elsewhere, we have decided to not add any CVE descriptions for 1.24, including this one. Removed.